src/core/plugin/luapi_application.h

```diff
@@ -625,9 +626,9 @@ static int applib_addSplines(lua_State* L) {
         UndoRedoHandler* undo = ctrl->getUndoRedoHandler();
         for (Element* element: strokes) undo->addUndoAction(std::make_unique<InsertUndoAction>(page, layer, element));
     } else if (strcmp("none", allowUndoRedoAction) == 0)
-        g_warning("Not allowing undo/redo action.");
+        return luaL_error(L, "Not allowing undo/redo action.");
```

Comment:
> to inform the plugin writer or user in case something may look unexpected

Yes I see this error/warning is not that severe. My thought was that as far as I've seen it, `g_warning` is only printed to the console and the plugin doesn't notice this. Therefore I thought, by making this an error, the plugin is able to notice and handle this.